distribution/docs/src/main/resources/content/_configuring/http-port.adoc

```diff
@@ -1,17 +1,23 @@
-:title: Configuring HTTP Port
+:title: Configuring HTTP(S) Ports
 :type: configuration
 :status: published
 :parent: Configuring Federation
-:summary: Configure HTTP Port
+:summary: Configure HTTP(S) Port.
```

Comment:
Small nit: Plural "ports" in title vs summary

Reply:
fixed.